internal/util/testing/testing.go

```diff
@@ -107,6 +108,10 @@ func NewTestInstance(
 	oc := config.Origins["default"]
 	p := NewTestPathConfig(oc, DefaultPathConfigs, urlPath)
 
+	tracer, _, _ := tr.Init(oc.TracingConfig)
+	// TODO worry about running closures for cleanup once the test is complete
```

Comment:
Is still to do?

Reply:
This is a to-do we'll leave for now since it's non-consequential to the running app or tests, but would be good to put in for completeness. My goal will be to get back to it some in the distant future but not in this PR.